crates/astria-conductor/src/executor/client.rs

```diff
@@ -68,7 +74,18 @@ impl Client {
         transactions: Vec<Vec<u8>>,
         timestamp: Timestamp,
     ) -> eyre::Result<Block> {
-        let transactions = transactions.into_iter().map(Bytes::from).collect();
+        use prost::Message as _;
+        let transactions = transactions
+            .into_iter()
+            .filter_map(|tx| match RollupData::decode(tx.as_slice()) {
```

Comment:
Should this actually throw an error instead of skipping?

Reply:
yeah, failing to decode here means there is an issue creating the `SequencerBlock` type in the sequencer